plugins/redis-lettuce/src/main/java/com/navercorp/pinpoint/plugin/redis/lettuce/interceptor/LettuceMethodInterceptor.java

```diff
@@ -41,19 +41,20 @@ public void doInBeforeTrace(SpanEventRecorder recorder, Object target, Object[]
     }
 
     @Override
-    public void doInAfterTrace(SpanEventRecorder recorder, Object target, Object[] args, Object result, Throwable throwable) {
+    public void doInAfterTrace(SpanEventRecorder recorder, Object target, Object[] args, Object result,
+                               Throwable throwable) {
         final String endPoint = toEndPoint(target);
         recorder.recordApi(getMethodDescriptor());
         recorder.recordEndPoint(endPoint != null ? endPoint : "Unknown");
-        recorder.recordDestinationId(LettuceConstants.REDIS_LETTUCE.getName());
+        recorder.recordDestinationId(endPoint != null ? endPoint : LettuceConstants.REDIS_LETTUCE.getName());
```

Comment:
@messi-gao
Since host information is recorded in the endpoint field, it is better to keep the destination ID as REDIS_LETTUCE.
What do you think?

Reply:
@emeroad It's OK,I have changed it.